server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/DatabaseChangeRuntimeFlowableTask.java

```diff
@@ -203,26 +219,67 @@ private DatabaseChangeThread generateOdcAsyncTaskThread(Long taskId, DelegateExe
     }
 
     private void modifyTimeoutIfTimeConsumingSqlExists(DelegateExecution execution,
-            DatabaseChangeParameters parameters) {
-        if (!parameters.isModifyTimeoutIfTimeConsumingSqlExists()) {
+            DatabaseChangeParameters parameters, DialectType dialectType, Long creatorId) {
+        long autoModifiedTimeout = flowTaskProperties.getIndexChangeMaxTimeoutMillisecond();
```

Comment:
What will happen if exception occurs inner this method? For example, the SQL is invalid and exception will be thrown when parse SQL. I thought that the task will be failed but these error message may not displaied in the log file. So better to put these codes in a try-catch block. There are two ways:
1. Throw a exception and then abort the task;
2. Skip this operation and contine the task.

I prefer the second way.

Reply:
You are right. I would add try catch in this method. 
but parse SQL exception was caught  in `SqlCheckUtil#parseSingleSql`.